atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -204,37 +181,23 @@ private CASResult updateTimestampInDb(Client client, Column oldColumn, Long oldV
         }
     }
 
-    private Column makeColumnWithNewFormat(@NotNull Long idToUse, Long ts) {
-        if (ts == null) {
-            return null;
-        }
-        return makeColumn(PtBytes.toBytes(idToUse + "_" + ts));
+    private boolean sameIdOrStartUp(TimestampBoundStoreEntry timestampBoundStoreEntry) {
+        return id.equals(timestampBoundStoreEntry.getId())
+                || (startingUp && timestampBoundStoreEntry.getTimestamp() == currentLimit);
```

Comment:
_slightly_ misleading as it returns false if we're starting up, but the limit is not as expected. Maybe `sameIdOrConsistentStartUp`?